src/services/dataContext.js

```
@@ -31,15 +29,32 @@ export async function verifyLocalDataContext(dataVersionKey, currentVersion) {
 export class DataContext {
+  static PREFIX = 'dataContext_'
+
   context = null
   dataPromise = null
 
+  /**
+   * Static method to clear all DataContext instances from localStorage
+   */
+  static async clearAll() {
+    const cache = new LocalCache()
+    cache.getKeys(this.PREFIX).then(keys => {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)